libs/blocks/footer/footer.js

```diff
@@ -95,7 +95,7 @@ class Footer {
       const headings = column.querySelectorAll('h2');
       headings.forEach((heading) => {
         const titleId = heading.textContent.trim().toLowerCase().replace(/ /g, '-');
-        const title = createTag('a', {
+        const title = createTag('p', {
           class: 'footer-nav-item-title',
           role: 'button',
           'aria-expanded': this.desktop.matches,
```

Comment:
Setting it to a `p` tag with a role of `button` seems very strange. Is there a case where this needs to be clicked, like on mobile?

Reply:
In mobile, the titles are clickable to expand/collapse the subitems.

Since the aria role should not be button on desktop, what about using matchMedia to toggle the aria role and properties?  Then using `<p>` makes more sense.

Reply:
agree with the comments above, you'd probably also want to modify the role on a mediaChange within the setDesktopTitles and setMobileTitles methods. 🙌

Reply:
Sounds good, added set and remove attributes to setDesktopTitles and setMobileTitles methods so they are updated on mediaChange.